Copy-edit 'Keyring support' section (#795)
pypa/twine: docs/index.rst

``twine register``
^^^^^^^^^^^^^^^^^^

.. warning::

>   The ``register`` command is `no longer necessary if you are
>   uploading to pypi.org`_.  As such, it is `no longer supported`_ in `Warehouse`_
>   (the new PyPI software running on pypi.org). However, you may need this if you
>   are using a different package index.

Reviewer: ```suggestion
   The ``register`` command is `no longer necessary if you are
   uploading to pypi.org`_.  As such, it is `no longer supported`_ in `Warehouse`_
   (the new PyPI software running on pypi.org). However, you may need this if you
   are using a different package index.
```

>    The ``register`` command is `no longer necessary if you are
>    uploading to pypi.org`_.  As such, it is `no longer supported`_ in `Warehouse`_
>    (the new PyPI software running on pypi.org). However, you may need this if you
>    are using a different package index.

Reviewer 2: This change is technically out of scope, but I'll allow it. 😉  Might as well tidy it up:

```suggestion
   The ``register`` command is `no longer necessary if you are
   uploading to pypi.org`_.  As such, it is `not supported by Warehouse`_
   (the software running on pypi.org). However, you may need it if you
   are using a different package index.
```

Author: Maybe making [pypi.org](https://pypi.org/) clickable here will be better?

Reviewer 2: Good call. Probably better for the link to read PyPI. I tweaked my suggestion accordingly.

Author: Thanks!
